page.go

```diff
@@ -36,6 +36,8 @@ type Page struct {
 	windowObjectID      proto.RuntimeRemoteObjectID // used as the thisObject when eval js
 	getDownloadFileLock *sync.Mutex
 
+	viewport *proto.PageVisualViewport
```

Comment:
we can remove it since the `GetViewport` seems can always return the latest size?

Reply:
I guess so